### ISSUE TITLE
Ligatures for mac

### ISSUE THREAD

**Issue by argideritzalpea** [silnrsi/font-tagmukay#2]
Is this confirmed to work with a particular Mac-based word processor?

I have installed it with keyman and am using it with Libre Office and the ligatures don't seem to be working

**Comment by jcoblentz** [comment]
Yes, we have verified it works with LibreOffice version 6.3.3 for Mac. It should work in later versions also. In the zip file you downloaded, there is a "documentation" folder. Go there and open "Tagmukay-Glyphs.odt" in LibreOffice. The Tifinagh section of the document should look like this:
![Tagmukay glyphs](https://user-images.githubusercontent.com/12460265/79393334-5c308f00-7f32-11ea-866e-cd61d18e4e71.jpg)
If it does not look like this, then make sure you have at least version 6.3.3 (or later) of LibreOffice.

Also make sure you are typing the "Tifinagh Consonant Joiner" in between the consonants to make the ligature. (See the second paragraph of https://github.com/silnrsi/font-tagmukay/blob/master/documentation/Tagmukay-SmartFontFeatures.pdf for an example.) On the Tawallammat Tifinagh (SIL) keyboard, press both AltGr and è (for French keyboards) or "Right Alt" and 7 (for English keyboards) to type the Tifinagh Consonant Joiner.

As a test, you can also enter the Tifinagh Consonant Joiner in LibreOffice by typing U+2D7F and then pressing both Alt and X.

I hope this helps.

**Comment by argideritzalpea** [comment]
Thanks for your quick response. The Mac keyboard does not have a typical AltGr key, so I have tried every combination of "option", "shift", "command", and "control" keys along with 7 in order to type the consonant joiner, but it doesn't print out. Is there any further advice for getting this to work on a Mac keyboard?

I have attached a picture. I noticed that in addition to not being able to produce the consonant joiner on my keyboard, it appears the 'l's and 'n's don't tilt when they are in succession, as seen in the screenshot.
<img width="880" alt="Screen Shot 2020-04-15 at 10 25 00 PM" src="https://user-images.githubusercontent.com/8315310/79417922-39b96880-7f68-11ea-82ad-bef4045bb425.png">

Also, thanks for making this and releasing it!

**Comment by martinpk** [comment]
On macOS, you can enable Unicode hex input: open System Preferences > Keyboard > Input Sources, click the + symbol, select the _Other_ category and add Unicode Hex Input. Then enable _Show input menu in menu bar_. You should now be able to select the Unicode hex input keyboard from the menu bar, and hold the Option key while typing _2d7f_ to insert the Joiner.

**Comment by DavidLRowe** [comment]
I don't have access to a Mac to test this, but in theory you would use the right Option key with the "7" key (to correspond to RightAlt+7 on a Windows keyboard).

**Comment by argideritzalpea** [comment]
Hi @martinpk, thanks for the tip. The consonant joiner works in adjoining characters and creating the ligatures with the font. The problem is being able to type the joiner in a natural way from the Tagmukay keyboard on a Mac.

@DavidLRowe  I have actually tried your suggestion and it prints out a paragraph symbol: ¶

**Comment by DavidLRowe** [comment]
@argideritzalpea Have you been able to resolve your typing issues? If you still have issues with Keyman, you might consider posting your question on the Keyman Community Forum (https://community.software.sil.org/c/keyman). No guarantees of an answer, but there are some Mac users who follow the discussions and perhaps one can help.
